src/current/v23.1/monitoring-and-alerting.md

```diff
@@ -189,6 +189,685 @@ If you rely on external tools for storing and visualizing your cluster's time-se
+| `config.leasePreferences`      | `[...]`                                                      | The [lease preferences](configure-replication-zones.html#lease_preferences) for the range.                                     |
+| `config.rangefeedEnabled`      | `false`                                                      | Whether [rangefeeds](create-and-configure-changefeeds.html#enable-rangefeeds) are enabled for this range.                      |
+| `config.excludeDataFromBackup` | `false`                                                      | Whether this range's data should be excluded from [backups](backup.html).                                                      |
+
+#### Examples
+
+- [Replication status - normal](#replication-status-normal)
+- [Replication status - constraint violation](#replication-status-constraint-violation)
+- [Replication status - under-replicated ranges](#replication-status-under-replicated-ranges)
+- [Replication status - ranges in critical localities](#replication-status-ranges-in-critical-localities)
+
+##### Replication status - normal
+
+The following example assumes you are running a fresh, local [`cockroach demo`](cockroach-demo.html) cluster started using the following command:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+cockroach demo --geo-partitioned-replicas --insecure
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl -X POST http://localhost:8080/_status/critical_nodes
+~~~
+
+~~~ json
+{
+  "criticalNodes": [
+  ],
+  "report": {
+    "underReplicated": [
+    ],
+    "overReplicated": [
+    ],
+    "violatingConstraints": [
+    ],
+    "unavailable": [
+    ],
+    "unavailableNodeIds": [
+    ]
+  }
+}
+~~~
+
+{{site.data.alerts.callout_info}}
+You may have to wait a few minutes after starting the demo cluster before getting the 'all clear' output above. This can happen because it takes time for [replica movement](architecture/replication-layer.html) to occur to meet the constraints given by the [zone configurations](configure-replication-zones.html) set by the [`--geo-partitioned-replicas` flag](cockroach-demo.html#geo-partitioned-replicas).
+{{site.data.alerts.end}}
+
+##### Replication status - constraint violation
+
+The following example assumes you are running a fresh, local [`cockroach demo`](cockroach-demo.html) cluster started using the following command:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+cockroach demo --geo-partitioned-replicas --insecure
+~~~
+
+By default, this geo-distributed demo cluster will not have any constraint violations.
+
+To introduce a violation that we can then query for, we'll modify the [zone configuration](configure-replication-zones.html) of the [`movr.users`](movr.html#the-movr-database) table.
+
+Let's use [`SHOW CREATE TABLE`](show-create.html) to see what existing zone configurations are attached to the `users` table, so we know what to modify.
```

Comment:
```suggestion
You can use [`SHOW CREATE TABLE`](show-create.html) to see what existing zone configurations are attached to the `users` table, so that you know what to modify.
```

Reply:
updated!